pkg/testutils/builder/defaultcr.go

```diff
@@ -0,0 +1,64 @@
+package builder
+
+import (
+	"fmt"
+
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+type DefaultCRBuilder struct {
+	defaultCR *unstructured.Unstructured
+}
+
+const (
+	InitSpecKey   = "initKey"
+	InitSpecValue = "initValue"
```

Comment:
These are actually not used here and have no relation to the builder itself. Can you move them to the testcases that use them and do assertions with them?

Reply:
Or the `NewDefaultCRBuilder()` should initialize the internal `defaultCR *unstructured.Unstructured` Spec already with an init kvp?

Reply:
This is actually a problem of the existing test, this field you can see is defined in `operator_v1beta2_moduletemplate_kcp-module.yaml`, and we are using it across different test suites (kyma-controller, control-plane), I also have no good idea where to put it.

IMO the `NewDefaultCRBuilder` should not provide this spec field by default, it prevents us from having a test case which requires different `spec`.

Reply:
ok, split finally.